save-common/src/jvmMain/kotlin/org/cqfn/save/entities/BaseEntity.kt

```diff
@@ -0,0 +1,19 @@
+package org.cqfn.save.entities
+
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.MappedSuperclass
+
+/**
+ * base class for all entities
+ */
+@Suppress("USE_DATA_CLASS")
```

Comment:
Do we still complain on abstract classes?

Reply:
No, removed it